tutorials/plugins/editor/making_main_screen_plugins.rst

```diff
@@ -6,8 +6,8 @@ Making main screen plugins
 What this tutorial covers
 -------------------------
 
-As seen in the :ref:`doc_making_plugins` page, making a basic plugin that
-extends the editor is fairly easy. Main screen plugins allow you to create
+As seen in the :ref:`doc_making_plugins` page, you can make a basic plugin
+that extends the editor. Main screen plugins allow you to create
```

Comment:
```suggestion
Main screen plugins allow you to create
```
I think for this the paragraph flows better if the sentence is removed entirely.